jdk/test/sun/security/ssl/X509TrustManagerImpl/distrust/Entrust.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.io.File;
+import java.time.*;
+import java.util.*;
+import javax.net.ssl.*;
+
+/**
+ * @test
+ * @bug 8337664 8341059
+ * @summary Check that TLS Server certificates chaining back to distrusted
+ *          Entrust roots are invalid
+ * @library /lib/security
+ * @modules java.base/sun.security.validator
```

Comment:
Nit: `@modules java.base/sun.security.validator` is not needed in 8u. However, I see `@modules` occurrences in other 94 tests, are we keeping them on purpose to minimize the differences with newer releases?

NOTE: the same question applies to the next file, `Symantec.java`.

Reply:
Yeah, I tend to keep `@modules` lines. They don't do anything for JDK 8 so should be fine to keep.